docs/userguide/main.md

```diff
@@ -682,6 +682,13 @@ Internally you should use self-signed certificates (you can let the operator gen
 
 The operator contains several features that automate management tasks that might be needed during the cluster lifecycle. The different available options are documented here.
 
+### Cluster recovery
+
+This operator automatically handles common failure scenarios and restarts crashed pods, normally this is done in a one-by-one fashion to maintain quorum and cluster stability.
+In case the operator detects several crashed or missing pods (for a nodepool) at the same time it will switch into a special recovery mode and start all pods at once and allow the cluster to form a new quorum. This parallel recovery mode is currently experimental and only works with PVC-backed storage. If you encounter problems with it, you can disable it by redeploying the operator and adding `manager.parallelRecoveryEnabled: false` to your `values.yaml`. Please also report any problems by opening an issue in the operator github project.
+
```

Comment:
Maybe add also how we are doing it ?
And, do we return the podManagementPolicy to his original value after the cluster is back to normal ?

Reply:
I've extended the explanation a bit. 
And yes, as soon as all pods are running the podManagemtnPolicy is put back.